usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateAllOptionalFileName.java

```diff
@@ -0,0 +1,40 @@
+package org.mobilitydata.gtfsvalidator.usecase;
+
+import org.mobilitydata.gtfsvalidator.usecase.notice.OptionalFileFoundNotice;
+import org.mobilitydata.gtfsvalidator.usecase.port.GtfsSpecRepository;
+import org.mobilitydata.gtfsvalidator.usecase.port.RawFileRepository;
+import org.mobilitydata.gtfsvalidator.usecase.port.ValidationResultRepository;
+
+import java.util.List;
+
+public class ValidateAllOptionalFileName {
```

Comment:
I think this class should be renamed - it doesn't really validate all optional file names, but more discovers and adds optional files to the provided file list (if I'm understanding correctly looking at the code). Maybe `DiscoverAndAddOptionalFiles`?

Reply:
Given the code's behavior, I agree

Reply:
The intent here is to actually validate that all non required files names indeed are in the schema definition. Will make modifications as such.